torchvision/datasets/standford_cars.py

```diff
@@ -0,0 +1,128 @@
+import os
+import os.path
+from typing import Callable, Optional
+
+import scipy.io as io
```

Comment:
You need to import `scipy` lazily as it is only an optional dependency of `torchvision`.